### DISCUSSION TITLE
Edgy Groups are awesome!

### DISCUSSION THREAD

**Discussion by rocket-engineer** [lucobellic/edgy-group.nvim#32] [Show and tell]
This is my current edgy-group setup (C++ and GoogleTest example) and it becomes better and better!

Main group (Neotree and Aerial):
![edgy-group-1](https://github.com/user-attachments/assets/dc218a68-c063-451f-ae0a-7fb574d3c1a6)

Git group (Neotree and Neotree git_status):
![edgy-group-2](https://github.com/user-attachments/assets/d9c8e472-0ac8-4faa-9cd7-946fdebdb687)

Testing group (Neotree and Neotest summary):
![edgy-group-3](https://github.com/user-attachments/assets/acb19a8a-fd78-4509-baca-4709ebc94f35)

Debugging group (Neotree and DAP UI):
![edgy-group-4](https://github.com/user-attachments/assets/a37b35c1-87ca-44c7-941b-1620e9ddaa4c)

I recently discovered how to setup edgy and the edgy groups (empty titles list!) also for debugging:
![edgy-group-5](https://github.com/user-attachments/assets/dd18b6dc-032b-4668-8b0b-0318fd3088b9)

Further groups are planned for Diffview.nvim and Octo.nvim :)

**Comment by lucobellic** [comment]
Thanks for your feedback!
It's awesome to see these examples with screenshots and configuration.
Feel free to ask for any features you think might be missing in edgy-group.
